dist/screenfull.d.ts

```diff
@@ -12,6 +12,20 @@ declare namespace screenfull {
 
 	type EventName = 'change' | 'error';
 
+	interface RequestOptions {
```

Comment:
Does the DOM types expose a type we could use for this instead of having to make our own?